vcmiqt/MessageBox.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * MessageBox.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+
+#pragma once
+
+#include "vcmiqt.h"
+
+#include <QMessageBox>
+#include <QTimer>
+
+namespace MessageBox
+{
+#ifdef VCMI_IOS
+	// iOS can't display modal dialogs when called directly on button press
+	// https://bugreports.qt.io/browse/QTBUG-98651
```

Comment:
btw please use the new function in already existing places that have this comment